.travis.yml

```diff
@@ -60,6 +60,7 @@ addons:
   # for Mac builds, we use Homebrew
   homebrew:
     packages:
+      - bison
```

Comment:
are you sure that you pulled before this PR? This problem with bison was solved here:

https://github.com/BlueBrain/nmodl/pull/400

Reply:
You're right. I somehow missed some commits